### DISCUSSION TITLE
where are the save files located?

### DISCUSSION THREAD

**Discussion by guranu** [skyline-emu/skyline#1018] [Q&A] [answered]
i was planning on upadating skyline untill i realised that all of the progeress in Celeste would be gone so i was wondering where are the save files located?

**Comment by ghost** [comment, marked as answer]
Internal Storage/android/data/skyline.emu/a bunch of subfolders/save

**Comment by miyamoto67** [comment]
Una pregunta el nivel de *reflexión* del celeste la pantalla se pone en blanco como hago para quitarlo bueno si te a pasado

**Reply by guranu** [reply]
how times has it been said speak english not spanish.
